README.md

```diff
@@ -1,2 +1,143 @@
-# camel-dap-client-emacs
-Extension to leverage Debug Adapater for Camel in Emacs
+# EMACS Debug Adapter client for Apache Camel
+
+![A breakpoint hit on a Camel route endpoint](./images/Emacs-dap.gif)
+
+# How to debug Camel with emacs
+I just got the debug integration for camel in emacs running using (https://emacs-lsp.github.io/dap-mode/page/adding-debug-server/). In order to not forget the config, I decided to quickly compile this readme.
+
+## Prerequisites
+ - version 3.16+
+ - camel-debug is available on the classpath
+ - have JMX enabled
+
+# How to configure Emacs
+
+In a `~/.emacs.d/dap-camel.el` file:
+
+```
+(dap-register-debug-provider
+ "camel"
+ (lambda (conf)
+   (plist-put conf :host "localhost")
+   conf))
+
+(dap-register-debug-template "Camel Attach"
+                             (list :type "camel"
+                                   :request "attach"
+                                   :args ""
+				                   :port 1099
+				                   :dap-server-path '("java" "-jar" "PATH/TO/camel-dap-server-0.4.0.
+                                                       jar")
+                                   :name "Run Configuration"))
+
+(setq debug-on-error t)
+```
+<kbd>M-x</kbd> `eval-buffer`
+
+# How to get Camel Debug Adapter server.jar file?
+- I found 2 releases of camel DAP server here - https://github.com/camel-tooling/camel-debug-adapter/tags but it doesn't contain any jar files. To get a jar file either you can compile the source or download from the Maven Central repository.
```

Comment:
i think we can simplify by mentioning only to download from Maven central. There is no plan to provide the jar on GitHub release